Doc/c-api/structures.rst

```diff
@@ -70,6 +70,14 @@ the definition of all other Python objects.
       (((PyObject*)(o))->ob_type)
 
 
+.. c:function:: int Py_IS_TYPE(PyObject *o, PyTypeObject *type)
+
+   Return true if the type of object *o* is the *type*.
+   Equivalent to: Py_TYPE(o) == type.
+
+   .. versionadded:: 3.9
+
```

Comment:
Ooops, I noticed a typo in one of my previous commit. Would you mind to take this PR as an opportunity to fix it?

https://docs.python.org/dev/c-api/structures.html#c.Py_SET_SIZE

"Set the object o size of size." => "Set ... to size."

Reply:
Oh sure :)

Reply:
Wait, @brandtbucher wrote PR #18496. I will ask him to fix the typo there. You can ignore this special request ;-)

Reply:
Got it ;-)

Reply:
## Current
<img width="760" alt="스크린샷 2020-02-13 오후 11 23 37" src="https://user-images.githubusercontent.com/5110323/74444397-29c5cf80-4eb8-11ea-9421-715c3a677aea.png">

## Suggestion
<img width="826" alt="스크린샷 2020-02-13 오후 11 15 15" src="https://user-images.githubusercontent.com/5110323/74444400-2b8f9300-4eb8-11ea-89c9-9ffcf3615213.png">

Looks like not work as we want ;)